src/ecrc-frontend/src/components/page/bcscRedirect/BcscRedirect.stories.js

```diff
@@ -12,6 +14,14 @@ const page = {
 };
 
 storiesOf("BcscRedirect page", module)
-  .add("Default", () => <BcscRedirect page={page} />)
+  .add("Default", () => (
+    <MemoryRouter>
+      <BcscRedirect page={page} />{" "}
```

Comment:
what is `{" "}` for? do we need it here?

Reply:
sry, removed it.